### PR TITLE
feat(eventstore): Use nodestore for get_events

### PR DESCRIPTION
**Pull request by lynnagara**
Now that we are caching nodestore data, try using nodestore for
get_events again. This functionality is toggled by the "eventstore.use-nodestore" option.

### CODE REVIEW

**Review by fpacifici — COMMENTED** [review]
This is going to roll the change out in one shot. What was the impact on nodestore without cache? (which would be the worst case scenario). Or should we roll this change out gradually ?

**Comment by lynnagara** [comment]
Previously I think we saw the organization events endpoint response time increase to ~2-3 seconds on average. I figured we can revert relatively quickly if this doesn't work out.

File: src/sentry/options/defaults.py [hunk 1]
```diff
@@ -206,5 +206,8 @@
 register("nodedata.cache-sample-rate", default=0.0, flags=FLAG_PRIORITIZE_DISK)
 register("nodedata.cache-on-save", default=False, flags=FLAG_PRIORITIZE_DISK)
 
+# Use nodestore for eventstore.get_events
+register("eventstore.use-nodestore", default=False, flags=FLAG_PRIORITIZE_DISK)
```
Comment (fpacifici) [inline src/sentry/options/defaults.py]: Shouldn't this be registered in a separate PR

**Review by fpacifici — CHANGES_REQUESTED** [review]

File: src/sentry/options/defaults.py (hunk 1 above)
Comment (lynnagara) [reply src/sentry/options/defaults.py]: Yup, https://github.com/getsentry/sentry/pull/16686

**Review by fpacifici — APPROVED** [review]
